### PR TITLE
Zcu102 7.0.0

### PR DESCRIPTION
**Pull request by Furao**
This pull request adds AARCH32 and AARCH64 support for the Xilinx Zynq UltraScale+ MPSoC. Tested on the ZCU102 development board.

### CODE REVIEW

**Comment by Furao** [comment]
Feel free to drop the "Added DARPA License" commit. We just need to retain it in our own repositories.

File: libplatsupport/plat_include/zynqmp/platsupport/plat/timer.h [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        return 0;
+    }
+}
+
+int ttc_init(ttc_t *ttc, ttc_config_t config);
+int ttc_start(ttc_t *ttc);
+int ttc_stop(ttc_t *ttc);
+int ttc_set_timeout(ttc_t *ttc, uint64_t ns, bool periodic);
+void ttc_handle_irq(ttc_t *ttc);
+uint64_t ttc_get_time(ttc_t *ttc);
```
Comment (AdrianDanis) [inline libplatsupport/plat_include/zynqmp/platsupport/plat/timer.h]: bit of a nitpick, but given that these are an interface to 'mach' code, then the interface should be moved to a 'mach' header and not duplicated across the two zynq platforms.

Comment (Furao) [reply libplatsupport/plat_include/zynqmp/platsupport/plat/timer.h]: I have a fix for this that I will be pushing up next week.

**Comment by Furao** [comment]
Updated based on comments.

**Pull request merged**